test/LetsEncrypt.Azure.UnitTests/AzureKeyVaultTests.cs

```diff
@@ -60,13 +87,42 @@ public void MultipleCallsToPersistCertificatesToAzureKeyVaultDoesNotDuplicateSer
             Assert.Single(provider.GetServices<ICertificateSource>().OfType<AzureKeyVaultCertificateRepository>());
         }
 
+        [Fact]
+        public async Task ImportCertificateChecksDuplicate()
+        {
+            const string Domain1 = "github.com";
+            const string Domain2 = "azure.com";
+
+            var certclient = new Mock<CertificateClient>();
+            var secretclient = new Mock<SecretClient>();
+            var logger = new Mock<ILogger<AzureKeyVaultCertificateRepository>>();
+            var options = new Mock<IOptions<LetsEncryptOptions>>();
+
+            options.Setup(o => o.Value).Returns(new LetsEncryptOptions
+            {
+                DomainNames = new[] { Domain1, Domain2 }
+            });
+
+            var repository = new AzureKeyVaultCertificateRepository(certclient.Object, secretclient.Object, options.Object, logger.Object);
```

Comment:
```suggestion
            var repository = new AzureKeyVaultCertificateRepository(certclient.Object, secretclient.Object, options, NullLogger<AzureKeyVaultCertificateRepository>.Instance);
```
I typically just pass in NullLogger as I'm not really interesting in testing for logging statements, so the mock is not really needed.

Reply:
Fixing.